File: example/javalib/testing/3-test-deps/qux/test/src/qux/QuxTests.java
package qux;

import com.google.common.base.Ascii;
import org.junit.Test;

public class QuxTests {

  @Test
  public void simple() {
    baz.BazTestUtils.bazAssertEquals(Ascii.toLowerCase("XYZ"), Qux.getValue());
  }
}

File: example/kotlinlib/testing/2-test-deps/qux/test/src/qux/QuxTests.kt
package qux

import com.google.common.base.Ascii
import io.kotest.core.spec.style.FunSpec

class QuxTests :
    FunSpec({

        test("simple") {
            baz.BazTestUtils.bazAssertEquals(Ascii.toLowerCase("XYZ"), Qux.VALUE)
        }
    })
